### DISCUSSION TITLE
Made a CLI wallpaper slideshow builder

### DISCUSSION THREAD

**Discussion by ghost** [mate-desktop/mate-desktop#623] [Show and tell]
As title, made a barebones CLI tool for making an .xml slideshow;  am planning to see if I can integrate it as an option the Appearance Preferences window (on the Backgrounds tab obviously).
[https://github.com/theangrygrunt/XML-slideshow-builder](https://github.com/theangrygrunt/XML-slideshow-builder)

**Comment by lukefromdc** [comment]
I'm not sure how much of the MATE userbase would use a cli background tool
I will leave this decision to the other team members, but I don't see us implementing this in our
existing packages. Note that a simple slideshow background option already exists in mate-settings-daemon
and in caja, and mate-backgrounds included an example from which others can be made.

On the other hand, a gui frontend dialog for this integrated into the appearance/backgrounds capplet 
would be quite likely to get used and would represent a useful new capability to many users. The same
backend script would simply take the output from the dialog.

**Reply by ghost** [reply]
Hi, thanks for your reply. I originally did want to integrate it with the appearances/backgrounds window but at the time I had no idea what it was called in the MATE repositories...so I ended up just making the script as a cli tool, but thank you for telling me what the actual name of what that window is. Just forked the mate-control-center repo and will get to work.
